pkgs/applications/audio/lollypop/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchgit, meson, ninja, pkgconfig, wrapGAppsHook
+, appstream-glib, desktop-file-utils, gobjectIntrospection
+, python36Packages, gnome3, glib, gst_all_1 }:
+
+stdenv.mkDerivation rec  {  
+  pname = "lollypop";
+  version = "0.9.514";
+  name = "${pname}-${version}";
```

Comment:
`pname` is not used anywhere except here, maybe this could be `name = "lollypop-${version}";` instead?